### DISCUSSION TITLE
Drop the `rd.cos.X` parameters on 3.0

### DISCUSSION THREAD

**Discussion by Itxaka** [kairos-io/kairos#1322] [Ideas]
Currently we support both the old rd.cos.X and the newer rd.immucore.X parameters in the cmdline to configure several things under the immutable module (see more on https://github.com/kairos-io/immucore#kernel-configuration-parameters)

We should drop those under 3.0 as the newer ones should be used. Would drop a bit of code as well as a side effect.

**Comment by mauromorales** [comment]
Sounds good to me. Maybe we can already have a card to show a deprecation notice.
